static/app/views/organizationGroupDetails/eventQuickTrace.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import quickTraceExample from 'sentry-images/spot/performance-quick-trace.svg';
+
+import {promptsCheck, promptsUpdate} from 'app/actionCreators/prompts';
+import {Client} from 'app/api';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import {Panel} from 'app/components/panels';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Group, Organization, PromptActivity} from 'app/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import {snoozedDays} from 'app/utils/promptsActivity';
+import withApi from 'app/utils/withApi';
+
+type Props = {
+  api: Client;
+  group: Group;
+  organization: Organization;
+};
+
+type State = {
+  shouldShow: boolean | undefined;
+};
+
+class EventQuickTrace extends React.Component<Props, State> {
+  state: State = {
+    shouldShow: undefined,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  async fetchData() {
+    const {api, group, organization} = this.props;
+    const {project} = group;
+
+    const data = await promptsCheck(api, {
+      projectId: project.id,
+      organizationId: organization.id,
+      feature: 'distributed_tracing',
+    });
+
+    this.setState({shouldShow: this.shouldShow(data ?? {})});
+  }
+
+  shouldShow({snoozedTime, dismissedTime}: PromptActivity) {
+    if (dismissedTime) {
+      return false;
+    }
+    if (snoozedTime) {
+      return snoozedDays(snoozedTime) > 7;
+    }
+    return true;
+  }
+
+  trackAnalytics({eventKey, eventName}) {
+    const {group, organization} = this.props;
+    const {project} = group;
+
+    trackAnalyticsEvent({
+      eventKey,
+      eventName,
+      organization_id: parseInt(organization.id, 10),
+      project_id: parseInt(project.id, 10),
+      platform: project.platform,
+    });
+  }
+
+  handleClick({action, eventKey, eventName}) {
+    const {api, group, organization} = this.props;
+    const {project} = group;
+    const data = {
+      projectId: project.id,
+      organizationId: organization.id,
+      feature: 'distributed_tracing',
+      status: action,
+    };
+    promptsUpdate(api, data).then(() => this.setState({shouldShow: false}));
+    this.trackAnalytics({eventKey, eventName});
+  }
+
+  render() {
+    const {shouldShow} = this.state;
+    if (!shouldShow) {
+      return null;
+    }
+
+    return (
+      <StyledPanel dashedBorder>
+        <div>
+          <Description>
+            <h3>{t('Configure Distributed Tracing')}</h3>
+            <p>{t('See what happened right before and after this error')}</p>
+          </Description>
+          <ButtonList>
+            <Button
+              size="small"
+              priority="primary"
+              href="https://docs.sentry.io/product/performance/getting-started/"
```

Comment:
Could we use the platform to link directly to the right perf platform? eg. https://docs.sentry.io/platforms/javascript/performance/